common.yml

```diff
@@ -701,6 +701,43 @@ functions:
       content_type: application/octet-stream
       display_name: ${tool}
 
+  "create release artifacts":
+    - command: shell.exec
+      params:
+        working_dir: src/github.com/mongodb/mongo-tools
+        script: |
+          . ./set_goenv.sh
+          GOROOT="" set_goenv || exit
+          export EVG_PLATFORM=${_platform}
+          go run release/release.go build-archive
+    - command: archive.targz_pack
+      params:
+        target: src/github.com/mongodb/mongo-tools/upload.tgz
+        source_dir: src/github.com/mongodb/mongo-tools
+        include:
+          - "./mongodb-cli-tools-*"
+    - command: s3.put
+      params:
+        aws_key: ${aws_key}
+        aws_secret: ${aws_secret}
+        local_file: src/github.com/mongodb/mongo-tools/upload.tgz
+        remote_file: mongo-tools/pkgs/${build_id}/all-release-artifacts.tgz
+        content_type: application/x-gzip
+        bucket: mciuploads
+        permissions: public-read
+        display_name: All Release Artifacts (.tgz)
+    - command: s3.put
```

Comment:
Suggest moving this up above line 713 so it more directly follows the build, then have the "bundle everything, upload everything" commands follow, possibly with a comment (e.g. "for the notary task").

Reply:
Done!